qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -121,14 +121,16 @@
             no Windows
             gagent_check_type = get_interfaces
         - check_fsfreeze:
-            # fsfreeze series commands can't run on windows guest.
-            no Windows
             gagent_check_type = fsfreeze
             gagent_fs_test_cmd = "rm -f /tmp/foo; echo foo > /tmp/foo"
+            Windows:
+                gagent_fs_test_cmd = WIN_UTILS:\fsfreeze-write.py
+                freeze_timeout = 10
         - check_reboot_shutdown_fsfreeze:
             gagent_check_type = reboot_shutdown
         - check_snapshot:
             # fsfreeze series commands can't run on windows guest.
+            no Windows
```

Comment:
You also need `gagent_fs_test_cmd` for windows here, right?

Reply:
no, just disable Windows, this case isn't suitable for windows guest.
I am supposed to do it later.